src/components/GamesList.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+	<ol class="games" reversed>
```

Comment:
One thing we lose from our Jekyll version is the automatic list-of-games. I'm sure we could figure out a way to do it here, but it's a pretty low priority IMO. We aren't really dealing with such a massive catalog that maintaining the list ourselves is too hard.